package.json

```diff
@@ -32,6 +32,7 @@
     "raven-js": "3.14.0",
     "react": "15.4.2",
     "react-dom": "15.4.2",
+    "react-masonry-component": "^5.0.5",
```

Comment:
@ianb: do we have rules for including semver range operators?
It looks like all the other version numbers are using exact versions.

TL;DR: Drop the `^` here?